inst/WORDLIST

```diff
@@ -1,17 +1,20 @@
 Appsilon
 Boxifying
 ESLint
+GTM
```

Comment:
Please update the `WORDLIST` now that we don't need some of the words here.

Reply:
Cleaned up `WORDLIST`. Only `roxygen` should be added in the `WORDLIST`. Forgot to do this after updating `NEWS`.